Makefile

```diff
@@ -26,7 +26,7 @@ lint:
 
 .PHONY: unit-tests
 unit-tests:
-	pip uninstall -y prefect-saturn
+	pip uninstall -y prefect-saturn || true
```

Comment:
why was this necessary?

Reply:
breaks when prefect-saturn isn't already installed

Reply:
that's not the behavior I've seen

![image](https://user-images.githubusercontent.com/7608904/98140703-5cbb5380-1e8b-11eb-9c3d-bd7bc3d45807.png)

Reply:
I mean, I'm fine with this change. But I'm just trying to understand why the behavior is different in your environment than it is in mind, in case I run into this in the future.

This is my `conda info`

```text
     active environment : None
            shell level : 0
       user config file : /home/jlamb/.condarc
 populated config files : 
          conda version : 4.8.4
    conda-build version : not installed
         python version : 3.7.6.final.0
       virtual packages : __glibc=2.27
       base environment : /home/jlamb/miniconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/jlamb/miniconda3/pkgs
                          /home/jlamb/.conda/pkgs
       envs directories : /home/jlamb/miniconda3/envs
                          /home/jlamb/.conda/envs
               platform : linux-64
             user-agent : conda/4.8.4 requests/2.22.0 CPython/3.7.6 Linux/5.4.0-52-generic ubuntu/18.04.4 glibc/2.27
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False
```

Reply:
mine is `9.0.1`. I also have python 2.7 as my system default, so `make unittest` actually doesn't work for me anyways. Happy to change it back if you prefer.

Reply:
haha you should definitely not be using `python` 2.7 to do any local development, since all of our code is 3.7+

You don't have to revert this change, it's harmless. But I strongly recommend putting a Python from conda onto your PATH and not using Python 2.7 for anything

Reply:
Oh gods no, I don't _use_ it. It's just there :laughing: 

Using the default conda env is a good idea though